Base/usr/share/man/man5/presenter.md

```diff
@@ -27,6 +27,7 @@ Metadata consists of simple key-value pairs of properties, of which any may or m
 -   `color-scheme`: (string, currently `light` and `dark` are recognized) The color scheme of the presentation. Presenter uses this to determine which color(s) to use for black-out and white-out. Black-out always uses the background color of this color scheme and white-out uses the foreground color. The default is white.
 -   `width`: (float) One of the two required metadata properties. Determines the normative width of a slide in abstract units. Together with the aspect ratio, the height obtains a normative size and these two can then be used by slide objects for positioning. Note that because the displayed size of the slide will most likely not match these dimensions, the width is just important for positioning objects, not for determining how large the slide will appear. Dimensions are always translated to the actual displayed size.
 -   `aspect-ratio`: (string in the format `width:height`, e.g. `16:9`, width and height are integers) One of the two required metadata properties. Together with the width, determines the normative height of the slide for positioning. This is not the physical size in pixels; see the explanation on `width`.
+-   `footer-center`: (string with placeholders): Defines a custom footer text for Presenter to display. Depending on user settings, Presenter might show a different footer instead or might hide the footer altogether. The footer can use placeholders which are explained in more detail in [Presenter's documentation](help://man/1/Presenter).
```

Comment:
The placeholders aren't actually documented in Presenter(1), but please add them. :^)